google_configs/build_packages.sh

```diff
@@ -23,42 +24,56 @@ TIMESTAMP="$(date +%s)"
 function build_distro() {
   declare -r distro="$1"
   declare -r pkg_type="$2"
+  declare -r init_config="$3"
+  declare -r init_prefix="$4"
   declare files=("$@")
+  declare file_pattern='*[^.sh]'
+  declare init_files=(${init_config}/${file_pattern})
   declare name='google-config'
 
+  for ((i=0; i<${#init_files[@]}; i++)); do
+    file_name="$(basename ${init_files[$i]})"
+    file_entry="${init_config}/${file_name}=${init_prefix}/${file_name}"
+    init_files[$i]="$file_entry"
+  done
+
   if [[ "${pkg_type}" == 'deb' ]]; then
     name="${name}-${distro}"
   fi
 
   fpm \
     -s dir \
     -t "${pkg_type}" \
-    --description 'Google Compute Engine guest configs' \
+    --after-install "${init_config}/postinst.sh" \
+    --before-remove "${init_config}/prerm.sh" \
+    --depends "google-compute-engine-${distro}" \
+    --description 'Google Compute Engine Linux guest configuration' \
     --iteration "0.${TIMESTAMP}" \
     --license 'Apache Software License' \
     --maintainer 'gc-team@google.com' \
     --name "${name}" \
+    --replaces 'gce-startup-scripts' \
+    --replaces 'google-startup-scripts' \
     --rpm-dist "${distro}" \
     --url 'https://github.com/GoogleCloudPlatform/compute-image-packages' \
     --vendor 'Google Compute Engine Team' \
     --version '2.0.0' \
```

Comment:
The version bump should be a separate commit when we are ready to cut a release.